src/todoitems.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+class TodoItems extends React.Component {
+  render() {
+    const ids = Object.keys({ ...this.props.textLines });
```

Comment:
Again you've converted it to array. It's a nice trigger for rethinking your app design.